md_api/src/main/java/com/wipro/ats/bdre/md/api/TermStep.java

```diff
@@ -22,31 +22,31 @@
 import org.apache.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.config.AutowireCapableBeanFactory;
-import org.springframework.context.ApplicationContext;
-import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 /**
  * Created by arijit on 12/8/14.
  */
 public class TermStep extends MetadataAPIBase {
-    public TermStep() {
-        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
-        acbFactory.autowireBean(this);
-    }
 
     private static final Logger LOGGER = Logger.getLogger(TermStep.class);
     private static final String[][] PARAMS_STRUCTURE = {
             {"p", "sub-process-id", "Sub Process id of the step"}
     };
 
+    @Autowired
+    private StepDAO stepDAO;
+
+    public TermStep() {
+        AutowireCapableBeanFactory acbFactory = getAutowireCapableBeanFactory();
+        acbFactory.autowireBean(this);
+    }
+
     /**
      * This method calls TermStep proc which updates instance_exec and batch_consump_queue table.
      *
      * @param params String array having environment and process-id with their command line notations.
      * @return nothing.
      */
-    @Autowired
-    private StepDAO stepDAO;
 
     public TermStepInfo execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)